### PR TITLE
Fix/spot label

### PR DESCRIPTION
**Pull request by ValGeorgiev**
Fix comments label rendering on web

Tested on Fast 3G:
giphy (1)

### CODE REVIEW

**Comment by aronshamash** [comment]
@ValGeorgiev can we have a test for this ?

**Comment by nehasri89** [comment]
nice work! LGTM apart from the test as @aronshamash mentioned

File: packages/article-comments/__tests__/web/__snapshots__/shared.web.test.js.snap [hunk 1]
```diff
@@ -34,42 +35,23 @@ exports[`enabled comments 1`] = `
...
+
+exports[`enabled comments 1`] = `
+<div>
   <div />
 </div>
 `;
 
 exports[`single comment 1`] = `
 <div>
-  <p>
```
Comment (aronshamash) [inline packages/article-comments/__tests__/web/__snapshots__/shared.web.test.js.snap]: @ValGeorgiev is this expected ?

Comment (ValGeorgiev) [reply packages/article-comments/__tests__/web/__snapshots__/shared.web.test.js.snap]: @aronshamash yes, we don't render comment label, until they are presented

**Pull request merged**